src/AppDelegate.cpp

```diff
@@ -213,7 +213,7 @@ void AppDelegate::toggleDomainServer(bool start) {
 
 void AppDelegate::toggleAssignmentClientMonitor(bool start) {
     if (start) {
-        _acMonitorProcess->start(QStringList() << "--min" << "4");
+        _acMonitorProcess->start(QStringList() << "--min" << "5");
```

Comment:
The entity server fix was a race condition related to assignments being started and then "stopped" and then started again.... In the case of the messages-mixer, it would stop itself (sometimes) if the settings was empty. Since we didn't have settings, it would start the shut down process, and then get a new assignment (the entity server), but it would FINISH the shut down process for the messages-mixer while the entity server assignment was in the middle of starting. The result was that the assignment would have it's interested nodes list left in a corrupt/broken state.

Since it turns out we don't need settings on the message mixer, we just removed the code to get settings.